packages/esm-tb-app/src/config-schema.ts

```diff
@@ -37,8 +37,17 @@ export const configSchema = {
       monthOfTreatment: '1418AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
       adherenceAssessment: '164075AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
       nextAppointmentDate: '5096AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      tbOutcome: '159786AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      typeOfTb: '160040AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
```

Comment:
These 2 concepts have been duplicated. There's already `outcome` with the same concept and the `typeOfTb` looks similar to `site`